ecommerce/templates/mail/product_order_receipt/subject.txt

```diff
@@ -1 +1 @@
-Purchase Order Receipt
\ No newline at end of file
+MITx Online Purchase
```

Comment:
I looked through all the email receipts in my inbox, and this pattern seems most common:

```suggestion
Your receipt from MITx Online
```

If `MITx Online` is an available in an environment-specific setting string, we should use that. But not if it's more than 5m of effort.